src/vma/sock/sock-redirect.cpp

```diff
@@ -525,10 +525,8 @@ int vma_add_conf_rule(char *config_line)
 
 	int ret = __vma_parse_config_line(config_line);
 
-#if (VMA_OPTIMIZE_LOG <= 5)
 	if (*g_p_vlogger_level >= VLOG_DEBUG)
 		__vma_print_conf_file(__instance_list);
-#endif
 
```

Comment:
What is a reason it is changed?
See the same https://github.com/Mellanox/libvma/pull/350/files#diff-ba7a8ee0726e15a62de6713b16ff9cacR126